src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs

```diff
@@ -149,14 +152,30 @@ public class ImageSharpMiddleware
         /// <returns>The <see cref="Task"/>.</returns>
         public async Task Invoke(HttpContext context)
         {
-            IDictionary<string, string> commands = this.requestParser.ParseRequestCommands(context)
-                .Where(kvp => this.knownCommands.Contains(kvp.Key))
-                .ToDictionary(p => p.Key, p => p.Value);
+            var commands = this.requestParser.ParseRequestCommands(context);
```

Comment:
It's hard to tell what the returned type of `commands` is here, can we use the explicit declaration